shell/public/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Usage Analytics App",
+    "short_name": "UAA",
+    "description": "Gain insights into the usage of your DHIS2 instance",
```

Comment:
I think these hardcoded texts about the Usage Analytics App need to be replaced by something dynamic. Perhaps using info from `package.json`

Reply:
Hehe. Good catch.

Reply:
We could also remove it. I wonder how many users install our apps on their home screen, and we're not that focused on building PWA's either. Other than that I don't know if there's any benefit: https://developers.google.com/web/fundamentals/web-app-manifest

Reply:
One of our teams has built a prototype converting the Dashboards app to a PWA with pretty good success. We might end up using that style for some apps that make sense. We probably do not want people to run Maintenance app on their phones, but for a tablet light maintenance or viewing visualisations can make sense.

And, for external developers it is nice if we have built-in support for PWAs, so I do think having the option is nice.

Reply:
Good catch @HendrikThePendric 

I actually thought I had removed this file (I removed the link from index.html), which is why the UA info snuck in there.

I agree we should think about supporting mobile and PWA apps in the future, but I think I'd advocate for leaving out `manifest.json` until we have a solid strategy around that.  Additionally, we would need to create a whole templating system for static files in order to make the manifest.json data custom - CRA only supports %VAR% substitutions in `index.html`, so I don't think that's worth it at this point.  We currently generate `webapp.manifest` at build time, so we could in the future extend that to create `manifest.json` as well.

Anybody opposed to just leaving out `manifest.json` for now and adding it later?

Reply:
> And, for external developers it is nice if we have built-in support for PWAs, so I do think having the option is nice.

Yeah that sounds like a good reason.

Reply:
@amcgee, if it's in the name of scoping down: :+1:

Reply:
Doing it